core/query.py

```diff
@@ -91,11 +90,15 @@ def get_advanced_config_count_query(advanced_config, estimate_count):
     builder = AdvancedQueryBuilder()
     return builder.get_count_sql(advanced_config, estimate_count)
 
-def build_advanced_queries(advanced_config, limit=None, offset=None, estimate_count=None):
+def build_advanced_queries(advanced_config, limit=None, offset=None, estimate_count=False):
     count_sql = get_advanced_config_count_query(advanced_config, estimate_count)
+    if not count_sql and estimate_count: # if count returns 0, remove estimate TODO: figure out why count would contradict actual results
+        count_sql = builder.get_count_sql(advanced_config, False)
```

Comment:
There's no `builder` variable in this function
Probably meant `count_sql = get_advanced_config_count_query(advanced_config, False)`

Reply:
Spot on, pushed an update.